tests/lib/cli.js

```diff
@@ -145,10 +143,7 @@ describe("cli", () => {
             // Mock CWD
             process.eslintCwd = getFixturePath("configurations", "single-quotes");
 
-            assert.doesNotThrow(() => {
-                cli.execute(code);
-            });
-
+            cli.execute(code);
             cli.execute(code);
```

Comment:
This seems to me like the original `assert.doesNotThrow` call was redundant, so I think it can just be removed rather than duplicating this call.